recipes-wigwag/maestro/maestro_0.0.2.inc

```diff
@@ -0,0 +1,85 @@
+DESCRIPTION = "maestro is a runtime / container manager for deviceOS"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://src/${GO_IMPORT}/LICENSE;md5=1dece7821bf3fd70fe1309eaa37d52a2"
+
+GO_IMPORT = "github.com/PelionIoT/maestro"
+
+DEPENDS = "deviceos-users libuv"
+RDEPENDS_${PN} += "bash"
+
+# avoid the `-linkshared` option in this recipe as it causes a panic
+GO_LINKSHARED=""
+
+inherit pkgconfig systemd go gitpkgv edge
+
+RT_SERVICE_FILE = "maestro.service"
+PR = "r0"
+
+SRC_URI = "git://${GO_IMPORT};protocol=https \
+file://${RT_SERVICE_FILE} \
+file://maestro-watcher.service \
+file://maestro-watcher.path \
+file://maestro-config-uz.yaml \
```

Comment:
I think maestro-config-uz.yaml has been deleted. Possilby replaced by maestro-config.yaml

Reply:
Good catch @petedyerarm. I have fixed it.